wfe/web-front-end.go

```diff
@@ -539,6 +539,21 @@ func (wfe *WebFrontEndImpl) Challenge(authz core.Authorization, response http.Re
 }
 
 func (wfe *WebFrontEndImpl) Registration(response http.ResponseWriter, request *http.Request) {
+	if request.Method != "POST" {
+		wfe.sendError(response, "Method not allowed", http.StatusMethodNotAllowed)
```

Comment:
When you rebase this, be sure to pass the method they attempted as the details object.